cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -526,6 +526,10 @@ func TestSimulateTransactionError(t *testing.T) {
 	result := simulateTransactionFromTxParams(t, client, params)
 	assert.Greater(t, result.LatestLedger, int64(0))
 	assert.Contains(t, result.Error, "MissingValue")
+	require.Len(t, result.Events, 1)
+	var event xdr.DiagnosticEvent
+	require.NoError(t, xdr.SafeUnmarshalBase64(result.Events[0], &event))
+	fmt.Println(event)
```

Comment:
could you change the `fmt.Println` into `t.Log` ? 
i.e. so that in the success case, it won't write entries to the output.

Reply:
I will just remove the print altogether

Reply:
done, ptal